src/status_im2/contexts/quo_preview/preview.cljs

```diff
@@ -190,10 +190,10 @@
 (defn customization-color-option
   ([]
    (customization-color-option {}))
-  ([opts]
+  ([{:keys [feng-shui?] :as opts}]
    (merge {:key     :customization-color
            :type    :select
-           :options (->> colors/customization
+           :options (->> (merge colors/customization (when feng-shui? {:feng-shui nil}))
```

Comment:
@ilmotta I want to use our `preview/customization-color-option` in the `color` component preview. I want it to include `feng-shui` key. The value of the key is nil because there is not a particular hex color for `feng-shui`. It is being handled in code (a circle which is half white half black).

Reply:
I want `feng-shui` to be part of the colors menu :)

Reply:
Sorry to be pedantic @OmarBasem, but why in the colors menu if `feng-shui` is not a color? Doesn't seem correct to me. `feng-shui` is just a boolean property that tells if the yin/yang colors should be used.

Reply:
Hmm.. I will check with design team